src/Fingerprint.js

```diff
@@ -88,19 +88,17 @@ $.extend( SELF.prototype, {
 	},
 
 	/**
-	 * @param {string} languageCode
-	 * @param {wikibase.datamodel.Term} term
+	 * @param {wikibase.datamodel.Term} label
 	 */
-	setLabel: function( languageCode, term ) {
-		this._labels.setItem( languageCode, term );
+	setLabel: function( label ) {
+		this._labels.setItem( label.getLanguageCode(), label );
 	},
 
 	/**
-	 * @param {string} languageCode
 	 * @param {wikibase.datamodel.Term} label
 	 */
-	removeLabel: function( languageCode, label ) {
-		this._labels.removeItem( languageCode, label );
+	removeLabel: function( label ) {
```

Comment:
We also have `removeLabelFor` and `removeDescriptionFor` which do exactly that. The same arguments apply as above.